src/components/toniq-toggle-button/toniq-toggle-button.component.ts

```diff
@@ -0,0 +1,62 @@
+import {css, defineElementEvent, defineFunctionalElement, html} from 'element-vir';
+
+export const ToniqToggleButton = defineFunctionalElement({
+    tagName: 'toniq-toggle-button',
+    props: {
+        text: '',
+        checked: false,
```

Comment:
This isn't really _checked_ since it's not a _visually_ a checkbox, right?
```suggestion
        active: false,
```

Reply:
It is a checkbox visually there's just no _checkmark_ on the design but still considered checkbox.